jenkins/job-configs/kubernetes-jenkins-pull/clean-pr-node-e2e.yaml

```diff
@@ -9,6 +9,12 @@
             days-to-keep: 14
     wrappers:
         - e2e-credentials-binding
+        - workspace-cleanup:
+            dirmatch: true
+            exclude:
+            - 'k8s.io/test-infra/.git/'
+            - 'test-infra/.git/'
```

Comment:
the test-infra/ one is where we check out test-infra repo for bootstrap and clean-pr-node-e2e.sh.

The k8s.io/test-infra one is what clean-pr-node-e2e.sh uses.